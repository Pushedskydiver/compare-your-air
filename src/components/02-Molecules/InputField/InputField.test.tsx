import React from 'react';
import { cleanup, render } from '@testing-library/react';;
import InputFieldStyles from './InputField.styles';

afterEach(() => {
  cleanup();
});

describe('<InputfieldStyles />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <InputFieldStyles>
        Some content goes here.
      </InputFieldStyles>
    );
    expect(container).toMatchSnapshot();
  })
})