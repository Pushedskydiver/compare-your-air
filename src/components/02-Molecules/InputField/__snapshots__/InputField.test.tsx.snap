// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputfieldStyles /> should match snapshot 1`] = `
<div>
  <div
    class="sc-AxjAm kmcxnG"
  >
    Some content goes here.
  </div>
</div>
`;
